### PR TITLE
Softfail for bsc#1220722

### PR DESCRIPTION
**Pull request by foursixnine**

### CODE REVIEW

**Comment by foursixnine** [comment]
I don't have openqa clone thing in this machine, and I'm not sure the clonning with comments work for osd, @jlausuch can you give me a hand? https://openqa.suse.de/tests/14939950

cc @Martchus ^ would that work? if yes: where is it documented?

**Comment by jlausuch** [comment]
> I don't have openqa clone thing in this machine, and I'm not sure the clonning with comments work for osd, @jlausuch can you give me a hand? https://openqa.suse.de/tests/14939950
> 
> cc @Martchus ^ would that work? if yes: where is it documented?

with this branch: http://openqa.suse.de/tests/15006311

File: tests/microos/image_checks.pm [hunk 1]
```diff
@@ -33,6 +33,11 @@ sub run {
 
     # Verify that there is no unpartitioned space left
     my $left_sectors = (is_sle_micro("5.4+") && is_aarch64) ? 2048 : 0;
+    if (is_sle_micro("6.0") && is_aarch64 && get_var("HDD_1") =~ /qcow2/) {
```
Comment (jlausuch) [inline tests/microos/image_checks.pm:36]:
```suggestion
    if (is_sle_micro('>=6.0') && is_aarch64 && get_var("HDD_1") =~ /qcow2/) {
```

**Review by jlausuch — CHANGES_REQUESTED** [review]
It also happens in 6.1

**Comment by jlausuch** [comment]
New VR: https://openqa.suse.de/tests/15007273#step/image_checks/10

**Review by jlausuch — APPROVED** [review]
LGTM

**Pull request merged**